roles/edpm_libvirt/templates/qemu.conf.j2

```diff
@@ -1,7 +1,12 @@
 max_files = 32768
 max_processes = 131072
+{% if edpm_libvirt_tls_certs_enabled|bool %}
+vnc_tls = 1
+vnc_tls_x509_verify = 1
+{% else -%}
 vnc_tls = 0
 vnc_tls_x509_verify = 0
```

Comment:
can omit `vnc_tls_x509_verify`, that's already the default unless we set `default_tls_x509_verify`

Reply:
also isn't relevant when vnc_tls = 0